ddsrecorder_yaml/include/ddsrecorder_yaml/yaml_configuration_tags.hpp

```diff
@@ -44,6 +44,7 @@ constexpr const char* RECORDER_FILE_NAME_TAG("filename");
 constexpr const char* RECORDER_DOWNSAMPLING_TAG("downsampling");
 constexpr const char* RECORDER_BUFFER_SIZE_TAG("buffer-size");
 constexpr const char* RECORDER_EVENT_WINDOW_TAG("event-window");
+constexpr const char* RECORDER_LOG_PUBLISHTIME_TAG("log-publishTime");
```

Comment:
I know that the MCAP Message member is named `publishTime`, but it sounds weird to mix camelCase and snake case so I'd change `log_publishTime` to `log_publish_time` everywhere. Take also into account that we may want to extend to other databases in which publishTime is not defined

```suggestion
constexpr const char* RECORDER_LOG_PUBLISHTIME_TAG("log-publish-time");
```

Reply:
This can be an open discussion